packages/cli/src/lib/implementation/skip-plugins.utils.ts

```diff
@@ -0,0 +1,46 @@
+import chalk from 'chalk';
+import type { CategoryConfig, PluginConfig } from '@code-pushup/models';
+import { filterItemRefsBy, ui } from '@code-pushup/utils';
+
+export function validateSkipPluginsOption(
+  {
+    plugins,
+    categories,
+  }: {
+    plugins: PluginConfig[];
+    categories: CategoryConfig[];
+  },
+  {
+    skipPlugins = [],
+    verbose = false,
+  }: { skipPlugins?: string[]; verbose?: boolean } = {},
+): void {
+  const skipPluginsSet = new Set(skipPlugins);
+  const missingPlugins = skipPlugins.filter(
+    plugin => !plugins.some(({ slug }) => slug === plugin),
+  );
+
+  if (missingPlugins.length > 0 && verbose) {
+    ui().logger.info(
+      `${chalk.yellow(
```

Comment:
Not a change request just thoughts.

This message is quite similar to the one for the `onlyPlugins` arg. Would it make sense to reuse logic here?

Reply:
That's a great idea. I was trying to be as less invasive as possible, but this is a good approach, especially if we start adding more similar flags.

Reply:
I ment creating an custom error.

e.g.:

```ts
export class PluginFilterError extends Error {
  constructor(filterName: 'onlyPlugins' | 'skipPlugins') {
      super(`missingPlugins: ... for ${filterName}`); 
  }
}
```

Reply:
Oh, silly me. The logic was so similar I thought you referred to the whole file.

Reply:
Do you want me to rollback these changes?

Reply:
it ok. lets get it in